openapi/openapi.yaml

```diff
@@ -933,6 +933,46 @@ paths:
               schema:
                 $ref: "#/components/schemas/PythonPackageVersionsResponseError"
 
+  /python/package/versions/{os_name}/{os_version}/{python_version}:
```

Comment:
It might be a good idea to merge this with `/python/package/versions/` endpoint. It will be more unified - see #1547 

/close